model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -251,34 +251,45 @@ case class WorkspaceStatus(workspaceName: WorkspaceName, statuses: Map[String, S
 
 case class BucketUsageResponse(usageInBytes: BigInt)
 
-case class ErrorReport(source: String, message: String, statusCode: Option[StatusCode], causes: Seq[ErrorReport], stackTrace: Seq[StackTraceElement], exceptionClass: Option[Class[_]])
+case class CErrorReport(source: String, message: String, statusCode: Option[StatusCode], causes: Seq[CErrorReport], stackTrace: Seq[StackTraceElement], exceptionClass: Option[Class[_]])
```

Comment:
I don't like this name, especially since you used a suffix when choosing `ErrorReportBase` (presumably to make it easy to find when looking for things that start with "ErrorReport"). If changing `ErrorReport` to `RawlsErrorReport` is the right thing to do and avoids having to rename this, then I think it should be done. To make the review easier, you could do that rename as a separate PR ahead of the rest of these changes.